Source/Core/DiscIO/Enums.cpp

```diff
@@ -182,6 +182,8 @@ std::string GetSysMenuVersionString(u16 title_version)
   case Region::NTSC_K:
     region_letter = "K";
     break;
+  case Region::UNKNOWN_REGION:
```

Comment:
Shouldn't we put a LOG message for that case specifying the ```title_version```? We might also set that ```region_letter``` to something like "?" because it will be used later on and it will be strange to have a system menu version without the region letter, in my opinion.